dev-data/index.ts

```
@@ -1,24 +1,17 @@
+export type { AboutCourseInfo, DataMap } from './courses-data.types';
 export type {
   CourseTitle,
   ImageLink,
   MentorActivity,
   MentorshipCourse,
   MentorshipCourseRouteKeys,
   MentorshipDefaultRouteKeys,
-  MentorshipDetails,
   MentorshipRoute,
+  MentorshipDetailsType,
 } from './mentorship-data.types';
```

ESLint

[error] Expected MentorshipDetailsType before MentorshipRoute

(sort-exports/sort-exports)